frontend/src/routes/GymPage/index.tsx

```diff
@@ -0,0 +1,124 @@
+    width: 30px;
+    height: 30px;
+  }
+  margin-left: 0.5rem;
+  width: 30px;
+  height: 30px;
+`
+
+const GymBarContainer = styled.div`
+  position: sticky;
+  display: flex;
+  background-color: #fff;
+  align-items: center;
+  top: 0;
+`
+
+const DummyHistoryRow = styled.div`
+  justify-content: space-between;
+  text-align: center;
+  height: auto;
+  padding: 0.5rem;
+`
+
+export default function GymPage({ onLeftClick }: { onLeftClick?: () => void }) {
+  // const dispatch = useDispatch()
+  // const history = useHistory()
+  // const { gymStatus, gymHistory, isLoading } = useSelector((state: RootState) => state.gym)
+  // const [currentTab, setCurrentTab] = useState<number>(1)
+  // const sections = ['Gym', 'History']
+
+  // useEffect(() => {
+  //   dispatch(getGymStatus())
+  //   dispatch(getGymHistory())
+  // }, [dispatch])
+
+  // const content = () => {
+  //   let gymGroupArr: getGymHistory[] = gymHistory
+  //   if (currentTab === 2) {
+  //     gymGroupArr = gymStatus
+  //   }
+  //   const openOrPendingSupperGroups = gymGroupArr.filter(
+  //     (gymGroup : getGymHistory) =>
+  //       gymGroup.history
+  //   )
+
+  //   const nonActiveSupperGroups = gymGroupArr.filter(
+  //     (supperGroup: getGymHistory) =>
+  //       supperGroup.status !== SupperGroupStatus.OPEN && supperGroup.status !== SupperGroupStatus.PENDING,
+  //   )
+
+  const gymHistory = [
+    {
+      gymStatus: true,
+      keyStatus: 'cooltelegram',
+      requesttime: 1643388954,
+      telegramHandle: 'cooltelegram',
+      userID: 'A0123456Z',
+    },
+    {
+      gymStatus: true,
+      keyStatus: 'anothernewdude',
+      requesttime: 1643389801,
+      telegramHandle: 'anothernewdude',
+      userID: 'A0234567M',
+    },
+  ]
+
+  const [currentTab, setCurrentTab] = useState<number>(1)
+  const sections = ['Gym', 'History']
+  const gymTelegram = gymHistory.map((gym: any) => gym.telegramHandle)
+  const history = useHistory()
+  return (
+    <>
+      <GymBarContainer>
+        <NavBarIcons
+          type="left"
+          onClick={() => {
+            onLeftClick ? onLeftClick() : history.goBack()
+          }}
+        />
+        {sections.map((section, index) => {
+          const isSelected = sections.indexOf(section) === currentTab - 1
+          return (
+            <>
+              <GymTabContainer key={index} onClick={() => setCurrentTab(index + 1)} isSelected={isSelected}>
+                {section}
+              </GymTabContainer>
+            </>
+          )
+        })}
+      </GymBarContainer>
+      {currentTab === 1 ? (
+        <>
+          <GymStatus isOpen />
+          <GymKeyWith />
+        </>
+      ) : (
+        <>
+          {gymHistory.map((gymGroup, index) => {
+            return (
+              <DummyHistoryRow key={index}>
+                {gymGroup.telegramHandle}
+                {gymGroup.userID}
+              </DummyHistoryRow>
+            )
+          })}
+        </>
+      )}
+    </>
```

Comment:
can consider abstract to a new file, where each file is one "tab" (maybe gymAvailabilityTab, gymHistoryTab)

and use a new switch method instead of a ternary for scalability concerns.